charts/metrics-agent/templates/deployment.yaml

```diff
@@ -52,6 +53,14 @@ spec:
             {{- toYaml .Values.resources | nindent 12 }}
           livenessProbe:
             {{- toYaml .Values.livenessProbe | nindent 12 }}
+          {{- with .Values.volumeMounts }}
```

Comment:
You also tested applying the chart with no volumes and no volume mounts and there were no errors?

Reply:
Yes, tested this use-case as well.